library/std/src/sys/pal/unix/thread.rs

```diff
@@ -729,16 +729,6 @@ mod cgroups {
     }
 }
 
-#[cfg(all(
-    not(target_os = "linux"),
-    not(target_os = "freebsd"),
-    not(target_os = "hurd"),
-    not(target_os = "macos"),
-    not(target_os = "netbsd"),
-    not(target_os = "openbsd"),
-    not(target_os = "solaris")
-))]
-#[cfg_attr(test, allow(dead_code))]
 pub mod guard {
```

Comment:
Why does this still exist as dummy?

Reply:
It's removed in the next commit. This is just so that I can extract the second commit to a separate PR should the need arise.